lib/models/evalutate_form/pre_visit/generalForm_model.dart

```diff
@@ -191,9 +191,9 @@ class GeneralFormModel {
     _previousIllness = map['previousIllness'];
     _drugUsed = map['drugUsed'];
     _asaClass = map['asaClass'];
-    _bw = map['bw'];
-    _high = map['high'];
-    _weight = map['weight'];
+    _bw = map['weight'];
+    _high = map['height'];
+    _weight = map['oldWeight'];
```

Comment:
Why don't we use the same "height" variable name

Reply:
@PormpimonPang  Because the field from front-end is `high`.  All of the variables are for front-end's sake. idk Might be a little confusion if we change. what do u think by the way.

Reply:
@Thaweesak-sai  Because at first, I think we might get confused when using it in the future. But now I understand that all variables for the front-end only. 
Like this is ok.